examples/nodejs/aws/eventbridge/README.template.md

```diff
@@ -0,0 +1,52 @@
+# Welcome to Momento <-> Eventbridge Integration Example Project
+
```

Comment:
in this PR i still do not see any updates to the READMEs in the parent directories, to provide links to this new example.